src/providers/ipa/ipa_s2n_exop.c

```diff
@@ -395,10 +414,18 @@ static errno_t s2n_encode_request(TALLOC_CTX *mem_ctx,
                                  request_type,
                                  domain_name,
                                  req_input->inp.name);
+                info = talloc_asprintf(mem_ctx,
+                            "EXTDOM EXPO request: [%s] domain: [%s] name: [%s]",
+                            ipa_s2n_reqtype2str(request_type),
+                            domain_name, req_input->inp.name);
```

Comment:
Why does commit of this patch say "Add the server IPA address"?
It looks like none of info strings contain this?

Reply:
added IPA server info as well